### ISSUE TITLE
Not able to get detailed accessibility report

### ISSUE THREAD

**Issue by ShashwatDubey** [nature/pa11y#70]
I am using pa11y for accessibility audit on my web application and it generate a report which shows  following details - 
* code : WCAG2AA.Principle2.Guideline2_4.2_4_2.H25.2
* message : Check that the title element describes the document.
* type : notice

But i am not able to get details like where any particular accessibility error/warning exists in my html. 
May be i am not able to understand.
so, please help me on this or let me know if there is any option to customize reports to get complete details to resolve issues in html.

**Comment by rowanmanning** [comment]
Hi Shashwat, you should be able to get a HTML snippet for the error with the `json` reporter:

```
pa11y -r json example.com
```

If you don't see `html` properties in the response, then you may need to update pa11y – it's a recent addition.

**Comment by ShashwatDubey** [comment]
@rowanmanning  Thanks buddy. 
I have installed pa11y using npm, 
Though, i made some changes in js files directly and it works for me. But still do you know any command to update pa11y using npm ?

**Comment by rowanmanning** [comment]
@ShashwatDubey yes, you can update by running:

```
npm install -g pa11y
```

And check the version with:

```
pa11y -v
```

It should be `1.7.0` for the latest version.

**Comment by rowanmanning** [comment]
Closing for now, let me know if you still have this issue.
